### DISCUSSION TITLE
Kakariko Village

### DISCUSSION THREAD

**Discussion by JCR64** [nzp-team/nzportable#906] [Map Releases]
Another N64 remake, this time Kakariko Village from The Legend of Zelda: Ocarina of Time. Only tested on FTE/PC, so again no guarantees for other platforms.

Easter Eggs:

- Find the Cuccos to unlock Pack-a-Punch machine
- Reach round 20 for a buyable ending and some music

Credits:

- Nintendo

# [DOWNLOAD (PC)](https://archive.org/download/nzp_kakarikovillage_oot64_pc/kakarikovillage_oot64.zip)

# [DOWNLOAD (PSP)](https://archive.org/download/nzp_kakarikovillage_oot64_psp/kakarikovillage_oot64-PSP.zip)

![screenshot1-bright](https://github.com/user-attachments/assets/0d99a784-176a-44f2-9469-1f672a564606)

![screenshot2-bright](https://github.com/user-attachments/assets/94f4e458-0a27-42d3-bb79-cfe10d3a08a8)

# [Video Playthrough](https://www.youtube.com/watch?v=Xwmkg8Co1SA)

## Changelog

2024-09-22

Updated to v1.1 and added a download for a PSP version. The PSP version is mostly downgrades to get it to run (at least on PPSSPP) and still suffers from some graphical hiccups so not the recommended way to play but it's still fun!

v1.1 Changes:
- Reaching the secret area doesn't unlock a door anymore (the fence by a Cucco) as the waypoint issues were too annoying to deal with. Instead you can still reach it but you are forced off by a trigger_hurt.

PSP Edition:
- Replaced ambience .wav (too big) with a stock cricket sound
- Sprites adjusted for Quake palette
- Removed rotation details (graphical glitches and/or crashing)
- Sky changed to plain black (original was too big)
- Added /n to long messages that were getting cut off
- Small adjustment to first barrier to give zombies more space to jump over it
- .mp3 instead of .ogg
- Compressed and downmixed .wav files

**Comment by DiabloGane45** [comment]
Goddamn that's beautiful

**Comment by LegendDD44** [comment]
the zombies gather in the middle then the pathfinding bugs and make my fps go from 400 to 2. after i go down or near the middle it is back to normal.

**Comment by PHDFLOPPERGUY56** [comment]
None of cuccos are working on the pc version except for the first one by quick revive

**Reply by JCR64** [reply]
I've tested it on the latest version for Windows and it seems OK.

Here's a quick video showing them all:

https://github.com/user-attachments/assets/0c792634-f4b6-40b5-b275-1b7903a39c2b

**Reply by ToonLink0001** [reply]
i tried doing that but it did not work

**Comment by ToonLink0001** [comment]
this map is good!

**Comment by mmkkiinn** [comment]
This map is very cool i play the map without ani problem thanks 😘🥰🥰😍

**Reply by ToonLink0001** [reply]
how?

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as you have different downloads depending on the platform

**Comment by Vortex71121** [comment]
this map has horrible collision, at least on 3DS

**Comment by nonMat** [comment]
This was quite fun!
Visually it's very impressive!
<img width="1920" height="1080" alt="immagine" src="https://github.com/user-attachments/assets/24dc65d6-2237-47a3-8147-3d7be1bc6e29" />

If a version with zoning could come out it would be even better!
